subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1607,6 +1658,14 @@ static int init_reset(void)
 #endif /* CONFIG_BT_CTLR_PHY_CODED */
 #endif /* CONFIG_BT_CTLR_PHY */
 
+#if defined(CONFIG_BT_CTLR_SYNC_TRANSFER_RECEIVER)
+	default_mode     = 0;
+	/* Default 6 events */
+	default_skip     = 6;
+	default_timeout  = 10;
+	default_cte_type = 0;
```

Comment:
They are chosen by me and no default values are defined in the core spec.

Reply:
> They are chosen by me as no default values are defined in the core spec.

Would it make sense to make this Kconfig configurable? 

The spec does not suggest any initial values for PAST if this is not set explicitly by the host?

Reply:
The default values here are just dummy values as `default_mode` set to 0 means the PAST is disabled `BT_HCI_LE_PAST_MODE_NO_SYNC.` Hence, the host will need to subscribe to PAST using `bt_le_per_adv_sync_transfer_subscribe` in order for PAST to be enabled by changing the `default_mode` and thereby also the `default_skip`, `default_timeout` and `default_cte_type`.

It might therefore be better to set everything to zero instead.

Reply:
> It might therefore be better to set everything to zero instead.

If `bt_le_per_adv_sync_transfer_subscribe` is required to be called by the host, then there's no value to have default values here (and may be harmful as it indicates that _some_ values are set). I suggest to leave them all as `0`.

Reply:
Agree, done